Project.toml

```
@@ -0,0 +1,18 @@
+name = "FTPServer"
+uuid = "9c2b0ca7-0fb3-5789-8f1a-7604e426024c"
+
+[deps]
+Conda = "8f4d0f93-b110-5947-807f-2305c1781a2d"
+Memento = "f28f55f0-a522-5efc-85c2-fe41dfb9b2d9"
+PyCall = "438e738f-606a-5dbb-bf0a-cddfbfd45ab0"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+
+[compat]
+julia = "≥ 1.0.0"
```

```suggestion
julia = "1"
```
We can't claim it'll run on Julia 2+